src/prefect/orion/database/migrations/versions/postgresql/2023_01_08_180142_d481d5058a19_rename_worker_pools_to_work_pools.py

```diff
@@ -12,7 +12,7 @@
 
 # revision identifiers, used by Alembic.
 revision = "d481d5058a19"
-down_revision = "f7587d6c5776"
+down_revision = "9326a6aee18b"
```

Comment:
I think this is breaking a chain of migrations that users have already run? Should `9326a6aee18b` be the latest revision?

Reply:
It's possible this is out of date! I'll check this one.

Reply:
Ah I see, does that matter?